### PR TITLE
Style hotfix

### PR DESCRIPTION
**Pull request by jvoros**
Put box-sizing on everything. That's my own crutch but seemed to fix some oddities, like the creative commons positioning on mobile. Change if needed.

Not sure why the wrapping flexbox on header was causing problems.

### CODE REVIEW

File: app/assets/css/header.styl [hunk 1]
```diff
@@ -2,11 +2,14 @@
 
 .header
     display: flex
-    flex-wrap: wrap
+    flex-flow: row wrap
```
Comment (dsifford) [inline app/assets/css/header.styl]: No difference between the old line and this line

File: app/assets/css/main.styl [hunk 2]
```diff
@@ -1,5 +1,8 @@
 @require 'variables'
 
+*
```
Comment (dsifford) [inline app/assets/css/main.styl]: This is the most specific selector that exists. Any style on the site that requires `content-box` on anything will automatically be overwritten by this. Dangerous

File: app/components/Header.jsx [hunk 3]
```diff
@@ -27,7 +27,7 @@ export default class Header extends PureComponent {
 
   render() {
     return (
-      <div className="row row--stacked">
+      <div className="header__wrap">
```
Comment (dsifford) [inline app/components/Header.jsx]: This doesn't conform with BEM.

Comment (dsifford) [inline app/components/Header.jsx]: Just checked it out on safari. Just remove the class entirely and add a margin of `$block-margins` to the search div and it should be good

File: app/assets/css/main.styl (hunk 2 above)
Comment (jvoros) [inline app/assets/css/main.styl]: That's exactly what I want. The other way is counterintuitive. Imma even expand it per CSS-Tricks recs: https://css-tricks.com/international-box-sizing-awareness-day/

File: app/components/Header.jsx (hunk 3 above)
Comment (jvoros) [inline app/components/Header.jsx]: That's better

**Pull request merged**